update-rules.json

```diff
@@ -829,5 +828,286 @@
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "deluge": {
+    "url": "http://dev.deluge-torrent.org/wiki/Download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#wikipage > p:nth-child(2)",
+		"baselink": "http://download.deluge-torrent.org/windows/deluge-{{.version}}-win32-py2.7-0.exe"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+  "sharex": {
+    "url": "https://github.com/ShareX/ShareX/releases/latest",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='setup.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "lockhunter": {
+    "url": "http://lockhunter.com/download.htm",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": ".download_comment",
+		"baselink": "http://lockhunter.com/exe/lockhuntersetup_{{.version-}}.exe"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "obs-studio": {
+    "url": "https://obsproject.com/download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='Full-Installer.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "colemak": {
+    "url": "https://colemak.com/Windows",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='Caps-Lock-Unchanged.zip']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "handbrake": {
+    "url": "https://handbrake.fr/downloads.php",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='i686-Win_GUI.exe']"
+      },
+	  "x86_64":{
+        "rule_type": "css-link",
+        "selector": "a[href$='x86_64-Win_GUI.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "windirstat": {
+    "url": "https://windirstat.net/download.html",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "li.download",
+		"baselink": "https://www.fosshub.com/WinDirStat.html/windirstat{{.version_}}_setup.exe"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "sumatrapdf": {
+    "url": "https://www.sumatrapdfreader.org/news.html",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": ".content > b",
+		"baselink": "https://www.sumatrapdfreader.org/dl/SumatraPDF-{{.version}}-install.exe"
+      },
+	  "x86_64":{
+        "rule_type": "css-html-version",
+        "selector": ".content > b",
+		"baselink": "https://www.sumatrapdfreader.org/dl/SumatraPDF-{{.version}}-64-install.exe"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "kodi": {
+    "url": "https://kodi.tv/download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='Jarvis.exe']"
```

Comment:
Ah yes, that could work, I already created an advanced rule [dcb3a8f](https://github.com/guiweber/just-install-updater/pull/8/commits/dcb3a8f0785ab54d387640aeec060d3ab6a2e964) but you don't have to merge it - I will try your idea.